go/chat/identify.go

```diff
@@ -28,10 +29,18 @@ func NewIdentifyNotifier(g *libkb.GlobalContext) *IdentifyNotifier {
 		Contextified: libkb.NewContextified(g),
 		DebugLabeler: utils.NewDebugLabeler(g, "IdentifyNotifier", false),
 		identCache:   make(map[string]keybase1.CanonicalTLFNameAndIDWithBreaks),
+		storage:      storage.New(g, func() libkb.SecretUI { return DelivererSecretUI{} }),
 	}
 }
 
 func (i *IdentifyNotifier) Send(update keybase1.CanonicalTLFNameAndIDWithBreaks) {
+
+	// Send to storage as well (charge forward on error)
+	if err := i.storage.UpdateTLFIdentifyBreak(context.Background(), update.TlfID.ToBytes(), update.Breaks.Breaks); err != nil {
```

Comment:
Here is where we update the identify status of the TLF. This function is triggered anytime we run `CryptKeys`,